template/_package.json

```diff
@@ -50,7 +50,9 @@
     "nuxt-buefy": "^0.3.2"<% } else if (ui === 'iview') { %>,
     "iview": "3.1.5"<% } %><% if (axios === 'yes') { %>,
     "@nuxtjs/axios": "^5.3.6"<% } %><% if (pwa === 'yes') { %>,
-    "@nuxtjs/pwa": "^2.6.0"<% } %><% if (ui === 'tachyons') { %>,
+    "@nuxtjs/pwa": "^2.6.0"<% } %><% if (typescript === 'yes') { %>,
+    "@nuxt/typescript": "^2.5.1",
```

Comment:
This would be better to be in `devDependencies` and instead, adding `ts-node` and `typescript` to dependencies.